Cargo.toml

```diff
@@ -35,9 +35,38 @@ grin_wallet_impls = { path = "./impls", version = "5.2.0-alpha.1" }
 grin_wallet_libwallet = { path = "./libwallet", version = "5.2.0-alpha.1" }
 grin_wallet_controller = { path = "./controller", version = "5.2.0-alpha.1" }
 grin_wallet_config = { path = "./config", version = "5.2.0-alpha.1" }
-
 grin_wallet_util = { path = "./util", version = "5.2.0-alpha.1" }
 
+
+##### Grin Imports
+
+# For Release
+#grin_core = "4.0.0"
+#grin_keychain = "4.0.0"
+#grin_util = "4.0.0"
+#grin_api = "4.0.0"
+
+# For beta release
+
+# grin_core = { git = "https://github.com/mimblewimble/grin", tag = "v5.0.0-beta.2"}
+# grin_keychain = { git = "https://github.com/mimblewimble/grin", tag = "v5.0.0-beta.2" }
+# grin_util = { git = "https://github.com/mimblewimble/grin", tag = "v5.0.0-beta.2" }
+# grin_api = { git = "https://github.com/mimblewimble/grin", tag = "v5.0.0-beta.2" }
+
+# For bleeding edge
+ grin_core = { git = "https://github.com/mimblewimble/grin", branch = "master" }
```

Comment:
nitpick: there are spaces/indentation that could be removed in this and other toml files